packages/connect-webextension/src/contentScript.ts

```diff
@@ -15,7 +16,7 @@ channel.init().then(() => {
     // once script is loaded. send information about the webextension that injected it into the popup
     window.postMessage(
         {
-            type: 'popup-content-script-loaded',
+            type: POPUP.CONTENT_SCRIPT_LOADED,
```

Comment:
this slightly increases bundle size of content-script.js 🗡️ 
it would like take all the imports from popup.ts with it 
```
import { UI_EVENT } from './ui-request';
import type { TransportInfo } from './transport';
import type { ConnectSettings, SystemInfo } from '../types';
import type { MessageFactoryFn } from '../types/utils';
```

Reply:
After it's minified the impact on output size is totally negligible, only +24 bytes.